components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/test/java/org/wso2/carbon/apimgt/rest/api/store/mappings/CommentMappingUtilTestCase.java

```diff
@@ -19,8 +19,8 @@
  */
 package org.wso2.carbon.apimgt.rest.api.store.mappings;
 
-import org.testng.Assert;
```

Comment:
tests already converted to junit. Lets keep them as it is for now. TestNG dependency was added back so we can write testng tests aswell.